src/mcts/search.cc

```diff
@@ -461,23 +467,30 @@ EdgeAndNode Search::GetBestChildWithTemperature(Node* parent,
             root_limit.end()) {
       continue;
     }
+
+    if (edge.GetN() <= static_cast<unsigned int>(kMinimumTemperatureVisits)) {
+      accepted_edges.push_back(edge);
+    } else {
+      all_edges.push_back(edge);
+    }
+  }
+
+  if (accepted_edges.empty()) {
+    accepted_edges = all_edges;
+  }
+
```

Comment:
you can move cummulative_sums declaration here and directly specify the size to the constructor

Reply:
can also move sum and n_parent here

Reply:
Just when I thought I was done... 


I *cannot* specify the size in the constructor, but I can reserve a size, which should be good enough.


Changes done, will push soon. Thank you again for checking these things.